tests/functional/simple_seed/test_seed.py

```diff
@@ -1,19 +1,278 @@
+            "seed_disabled.csv": seeds__disabled_in_config,
+            "seed_tricky.csv": seeds__tricky,
+        }
+
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "seeds": {
+                "test": {"seed_enabled": {"enabled": True}, "seed_disabled": {"enabled": False}},
+                "quote_columns": False,
+            },
+        }
+
+    @pytest.fixture(scope="function")
+    def clear_test_schema(self, project):
+        yield
+        project.run_sql(f"drop schema if exists {project.test_schema} cascade")
+
+    def test_simple_seed_with_disabled(self, clear_test_schema, project):
+        results = run_dbt(["seed"])
+        len(results) == 2
+        check_table_does_exist(project.adapter, "seed_enabled")
+        check_table_does_not_exist(project.adapter, "seed_disabled")
+        check_table_does_exist(project.adapter, "seed_tricky")
+
+    def test_simple_seed_selection(self, clear_test_schema, project):
+        results = run_dbt(["seed", "--select", "seed_enabled"])
+        len(results) == 1
+        check_table_does_exist(project.adapter, "seed_enabled")
+        check_table_does_not_exist(project.adapter, "seed_disabled")
+        check_table_does_not_exist(project.adapter, "seed_tricky")
+
+    def test_simple_seed_exclude(self, clear_test_schema, project):
+        results = run_dbt(["seed", "--exclude", "seed_enabled"])
+        len(results) == 1
+        check_table_does_not_exist(project.adapter, "seed_enabled")
+        check_table_does_not_exist(project.adapter, "seed_disabled")
+        check_table_does_exist(project.adapter, "seed_tricky")
+
+
+class TestSeedParsing(SeedConfigBase):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"seed.csv": seeds__wont_parse}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"model.sql": models__from_basic_seed}
+
+    def test_dbt_run_skips_seeds(self, project):
+        # run does not try to parse the seed files
+        len(run_dbt()) == 1
+
+        # make sure 'dbt seed' fails, otherwise our test is invalid!
+        run_dbt(["seed"], expect_pass=False)
+
+
+# BOM = byte order mark
+class TestSimpleSeedWithBOM(SeedConfigBase):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+
+    def test_simple_seed(self, project):
+
+        shutil.copyfile(
+            project.test_data_dir / Path("seed_bom.csv"),
+            project.project_root / Path("seeds") / Path("seed_bom.csv"),
+        )
```

Comment:
@gshank  this is currently being used to copy that file over to test dir. I wonder which part actually uses `pytest_csv`

Reply:
It looks like pytest-csv is for creating test reports in csv format. So I have no idea how it got involved, but I saw it in the error log.